auth0_flutter_platform_interface/lib/src/auth0_flutter_web_platform.dart

```diff
@@ -15,6 +15,10 @@ abstract class Auth0FlutterWebPlatform extends PlatformInterface {
     _instance = instance;
   }
 
+  Future<Object?> get appState {
```

Comment:
Previously there was also an `@override` that duplicated this implementation. I removed that one.
Maybe that is why the code coverage report was reporting the uncovered lines?